src/utils/change-case.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Converts a string to camelCase format
+ *
+ * @param {string} str String to be converted
+ * @returns converted string
+ */
+export const toCamelCase = (str: string) => {
+  return str.replace(/([-_\s]+[a-z])/gi, $1 => {
+    return $1
+      .toUpperCase()
+      .replace(/-/g, '')
```

Comment:
Would this work instead: `$1.substr(-1).toUpperCase()` ?

Reply:
(applies to the whole `return` statement, not just this line)

Reply:
I'll give it a try

Reply:
Removed in favour of lodash